components/transaction-chart/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Line } from 'react-chartjs-2';
+import { useState, useEffect } from 'react/cjs/react.development';
+import styles from './transaction-chart.module.css';
+const TransactionChart = (props) => {
+  const [chartData, setChartData] = useState({});
+
+  const chart = () => {
+    setChartData({
+      labels: [
+        '09/23/2020',

```

Comment:
Please shift this data into mock that would be easy when we will refactor code base with actual api.

Reply:
Sure,I will definitely do that and will update